src/OpenTelemetry.Api/Trace/Link.cs

```diff
@@ -42,7 +42,7 @@ public Link(in SpanContext spanContext)
         /// <param name="attributes">Link attributes.</param>
         public Link(in SpanContext spanContext, SpanAttributes attributes)
         {
-            this.ActivityLink = new ActivityLink(spanContext.ActivityContext, attributes.Attributes);
+            this.ActivityLink = new ActivityLink(spanContext.ActivityContext, attributes?.Attributes ?? null);
```

Comment:
`?? null` is basically a noop :smile:

`new ActivityLink(spanContext.ActivityContext, attributes?.Attributes);` should accomplish the same.